RELEASE_NOTES.md

```diff
@@ -52,13 +37,12 @@ $ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/lib
 ```
 
 - Wait for software upgrade proposal to pass and trigger the chain upgrade.
-- Run latest Price Feeder (v2.0.3) - **updated**
```

Comment:
Let's keep a note about the minimum price feeder version which has to be used. People are more looking into the release notes than the readme updates.

Reply:
Added back near this line (there is also a larger note about price feeder around line 17)